app/services/movie_service.rb

```diff
@@ -2,8 +2,8 @@ class MovieService
   def self.top_forty
     results = []
     page = 1
-    url = "/3/discover/movie?api_key=#{ENV['MOVIE_API_KEY']}&#{language}&#{sort}&#{adult}&#{video}&page=#{page}"
     2.times do
+      url = "/3/discover/movie?api_key=#{ENV['MOVIE_API_KEY']}&#{language}&#{sort}&#{adult}&#{video}&page=#{page}"
```

Comment:
I am glad this was just an order of operations issue and we were checking in deployment